src/libraries/illink-sharedframework.targets

```diff
@@ -30,7 +30,7 @@
       <_RuntimePackSuppressionsXmls Include="$(ILLinkTrimAssemblyRuntimePackSuppressionsXmlsDir)*.xml" />
 
       <!-- Collate CoreLib suppression XML files not bin-placed in earlier per-library linker run. CoreLib doesn't use bin-place logic. -->
-      <_RuntimePackSuppressionsXmls Include="$(CoreLibSharedDir)ILLink\ILLink.Suppressions.LibraryBuild.xml" />
+      <_RuntimePackSuppressionsXmls Condition="'$(TargetsMobile)' == 'true'" Include="$(CoreLibSharedDir)ILLink\ILLink.Suppressions.LibraryBuild.xml" />
```

Comment:
This isn't going to work. This line is for CoreLib, but the file in question is for System.Net.Http.

My thinking was to rename https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/ILLink/ILLink.Suppressions.LibraryBuild.xml to https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/ILLink/ILLink.Suppressions.Mobile.LibraryBuild.xml

And then in `System.Net.Http/src/System.Net.Http.csproj` add:

```xml
<ItemGroup>
  <ILLinkSuppressionsXmls Condition="'$(TargetsMobile)' == 'true'" Include="$(ILLinkDirectory)ILLink.Suppressions.Mobile.LibraryBuild.xml" />
```